src/backend/cdb/cdbllize.c

```diff
@@ -479,7 +479,8 @@ ParallelizeCorrelatedSubPlanMutator(Node *node, ParallelizeCorrelatedPlanWalkerC
 		 * unnest(array[typoutput, typsend]) from pg_type) then 'upg_catalog.'
 		 * else 'pg_catalog.' end) FROM pg_proc p;
 		 **/
-		if (scanPlan->flow && (scanPlan->flow->locustype == CdbLocusType_Entry))
+		Assert(scanPlan->flow);
+		if (scanPlan->flow->locustype == CdbLocusType_Entry)
```

Comment:
Pointer scanPlan->flow cannot be NULL here because it may be dereferenced [below](https://github.com/arenadata/gpdb/pull/752/files#diff-9885faa7c9bcd453dead0bb11b9d230cfa456f81c8faeb22c825624fe8d1dfbdR587-R601).

Reply:
I suppose you meant that it _must not_ be null rather that it can't